api/modules/automations.js

```diff
@@ -1,8 +1,105 @@
+const moment = require('moment')
+
 const github = require('../handlers/github')
 const { GITHUB } = require('../config/credentials')
+const db = require('../models')
 
 const automations = module.exports = (() => {
 
+    const createClient = async ({ clientInformation }) => {
+        const client = await getClientFromExternalId({ id: clientInformation.external_uuid })
+        const email = await getClientFromEmail( { email: clientInformation.email })
+        if (!client && !email) {
+            return db.models.Client.create({
+                email: clientInformation.email,
+                currency: clientInformation.currency,
+                name: clientInformation.name,
+                is_active: 1,
+                created_at: moment(clientInformation.date_created),
+                updated_at: moment(clientInformation.date_created),
+                external_uuid: clientInformation.external_uuid
+            })
+        } else {
+            updateClient({ clientInformation: clientInformation})
+        }
+    }
+
+    const updateClient = async (params) => {
+        let clientToUpdate
+        if (params.clientInformation.external_uuid){
+            clientToUpdate = await db.models.Client.findOne({
+                where: {
+                    external_uuid: params.clientInformation.external_uuid
+                }
+            })
+        }
+        if (params.clientInformation.email && (clientToUpdate === null)){
+            clientToUpdate = await db.models.Client.findOne({
+                where: {
+                    email: params.clientInformation.email
+                }
+            })
+        }
+        if (clientToUpdate) {
+            return db.models.Client.update({
+                email: params.clientInformation.email,
+                currency: params.clientInformation.currency,
+                name: params.clientInformation.name,
+                updated_at: moment(params.clientInformation.date_created),
```

Comment:
`updated_at` should be a current timestamp instead of the date created from Stripe

- Check if removing this causes Sequelize to update this automatically, if not use moment to get the current timestamp